Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -755,6 +825,11 @@ void CMultiplayerSA::InitHooks()
     HookInstall(HOOKPOS_CAnimManager_AddAnimationAndSync, (DWORD)HOOK_CAnimManager_AddAnimationAndSync, 10);
     HookInstall(HOOKPOS_CAnimManager_BlendAnimation_Hierarchy, (DWORD)HOOK_CAnimManager_BlendAnimation_Hierarchy, 5);
 
+    HookInstallCall(0x704EE8, (DWORD)HOOK_GrainEffect<GrainEffect::NightModifier>);
```

Comment:
The memory adresses here (and elsewhere) should really be constants somewhere. (similar to the HOOKPOS constants, although I'd put them into the GrainEffect namespace), to make it a bit clearer what we're actually hooking here